_layouts/sidebar.html

```diff
@@ -2,36 +2,37 @@
 layout: default
 ---
 
-{% assign toplevels = site.collections| where_exp: "item", "item.sidebar_index" | sort:"sidebar_index" %}
+{% assign toplevels = site.collections | where_exp: "item", "item.sidebar_index" | sort:"sidebar_index" %}
 <div class="container" style="padding-top: 24px;">
   <div class="row">
-    <div class="col-md-3" id="toc" style="padding-top: 16px;">      
-{% for group in toplevels %}
-    {% assign frontdoc=group.docs| where_exp: "doc","doc.path == group.frontpage"|first %}
-    {% unless frontdoc %}
-      {% assign frontdoc=group.docs[0] %}      
-    {% endunless  %}
-  {% capture target %}{% include link.html url=frontdoc.url %}{% endcapture %}
-    <div class="list-group">
-      <a class="list-group-item active" href="{{ target |strip }}">{{ group.caption }}</a>
-  {% if page.collection == group.label  %}
-    {% for doc in group.docs %}
-      {% assign classmodifier="" %}
-      {% if doc.url == page.url %}
-        {% assign classmodifier="disabled" %}
-	{% endif %}
-	{% capture target %}{% include link.html url=doc.url %}{% endcapture %}
-        <a class="list-group-item {{classmodifier}}" href="{{ target |strip }}">{% include title.html doc=doc coll=group %}</a>
-    {% endfor %}
-  {% endif %}
+    <div class="col-md-3" id="toc" style="padding-top: 16px;">
+    {% for group in toplevels %}
+      {% assign frontdoc=group.docs| where_exp: "doc","doc.path == group.frontpage"|first %}
+      {% unless frontdoc %}
+        {% assign frontdoc=group.docs[0] %}
+      {% endunless  %}
+      {% capture target %}{% include link.html url=frontdoc.url %}{% endcapture %}
+      <div class="list-group">
+        <a class="list-group-item active" href="{{ target |strip }}">{{ group.caption }}</a>
+        {% if page.collection == group.label  %}
+          {% assign subpages = group.docs | sort_natural:"title" | sort:"sort_key" %}
```

Comment:
Is the idea to sort by "sort_key" if available? If two pages have the same sort_key, their order will be unspecified, IMO. Currently they seem to be sorted by their title, but I'm not sure if it's guaranteed.

Reply:
Fixed